docs/react/getting-started.en-US.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 1
+title: Getting Started
+---
+
+Ant Design React is dedicated to providing a **good development experience** for programmers.
+
+---
+
+You had better learn [React](http://facebook.github.io/react/) and [ES2015](http://babeljs.io/docs/learn-es2015/) first.
+
+## First Example
+
+The following CodePen demo is the simplest usage case, and it's also a good habit to fork this demo to provide a re-producible demo while reporting a bug. Please don't use this demo as a scaffold in real world.
+
+- [antd CodePen](http://codepen.io/anon/pen/wGOWGW?editors=001)
+
+## Standard Development Flow
+
+You may need to compile and debug ES2015 and JSX, and proxy and deployment in a project.
```

Comment:
During development, you may need to compile and debug JSX and ES2015 code, and even proxy some of the request to mocked data or some external services. And all of these to be done with a quick feedback provided through hot reloading of changes.
Such features, together with packaging the production version are covered in this work flow.